docs/cluster-agent/CUSTOM_METRICS_PROVIDER.md

```diff
@@ -1,6 +1,6 @@
-# Datadog Cluster Agent - DCA | Horizontal Pod Autoscaling
+# Datadog Cluster Agent | Custom & External Metrics Provider 
 
-The DCA is a **beta** feature, if you are facing any issues please reach out to our [support team](http://docs.datadoghq.com/help)
+The Datadog Cluster Agent is a **beta** feature and the Custom Metrics Provider is in **alpha** if you are facing any issues please reach out to our [support team](http://docs.datadoghq.com/help).
```

Comment:
We should come up with a single term to mean `Custom & External Metrics Provider`

Sometimes we just refer to this as `HPA` and other places we use `Custom Metrics Provider` to refer to both.

What about `Custom Metrics Server` (I think that is the terminology Kubernetes uses)?

We can quickly explain that the `Custom Metrics Server` contains the `Custom & External Metrics Provider` and what the role of each one is. Then we can use those terms freely.

Reply:
yes agreed.